lib/wallet/wallet.js

```diff
@@ -1330,7 +1538,7 @@ class Wallet extends EventEmitter {
     if (tx.getSigopsCost(mtx.view) > policy.MAX_TX_SIGOPS_COST)
       throw new Error('TX exceeds policy sigops.');
 
-    if (tx.getWeight() > policy.MAX_TX_WEIGHT)
+    if (tx.getSize() > policy.MAX_TX_SIZE)
```

Comment:
Why did you change this ?

Reply:
Oh shoot, that was an accident. I needed to change it for the estimates, but as a tx this should be fine. Good catch